### ISSUE TITLE
Create a section about using project management and collaboration tools

### ISSUE THREAD

**Issue by fayepal** [mozilla/open-leadership-training-series#78]
As there have been non-technical projects in the training, some people aren't sure about where they can use Github. My idea is to have sessions on online project management and collaboration tools/ strategies/ best practices.

**Comment by betatim** [comment]
You thinking of tools like the "Projects" tab on Github?

Could also point towards some of the github repositories used to organise events/meetings.

**Comment by fayepal** [comment]
Yes. However, I am thinking to move away from Github because it might be confusing to other people because of its format that is optimized for coding projects. I am thinking of using the very non-coder friendly tools like Trello and maybe making things in a bit old-fashioned way, so that contributors won't have to deal with the frustration of having a new thing to learn/ having to install and sign up to many services. This is for projects that would be needing participation of the neighborhood, people involved with policy, advocacy and the like.

**Comment by dbild** [comment]
@fayetandog keep me updated on this.  I'm planning to work on this sort of thing for my sprint project (specifically looking at non-Github tools for collaboration that teens could use for non-code projects) https://github.com/dbild/teen-open-leadership/blob/master/CONTRIBUTING.md
